model/patch/patch.go

```diff
@@ -183,6 +191,8 @@ type Patch struct {
 	// MergedFrom is populated with the patch id of the existing patch
 	// the merged patch is based off of, if applicable.
 	MergedFrom string `bson:"merged_from,omitempty"`
+	// LocalModuleIncludes is only used for CLI patches to store local module changes
+	LocalModuleIncludes []LocalModuleInclude `bson:"-"`
```

Comment:
can you include some info about why this doesn't need to be stored in the DB? "Doesn't need to be stored because this the parser project is finalized at patch creation" or something. (Similarly, can this be private?)

Reply:
updated the comment but cannot make private because it's referred to in a lot of places